src/components/data-table/ha-data-table-icon.ts

```diff
@@ -50,8 +50,8 @@ class HaDataTableIcon extends LitElement {
       outline: none;
       font-size: 10px;
       line-height: 1;
-      background-color: var(--simple-tooltip-background, #616161);
-      color: var(--simple-tooltip-text-color, white);
+      background-color: #616161;
+      color: white;
```

Comment:
Can we replace this with another variable we already use? Is it needed for darkmode?

Reply:
This is a self made tooltip for this component only. I migrate it to ha-tooltip